source/features/fix-sticky-file-header-tooltips.js

```diff
@@ -0,0 +1,21 @@
+import select from 'select-dom';
+
+const oppositeDirections = {
+	'tooltipped-n': 'tooltipped-s',
+	'tooltipped-ne': 'tooltipped-se',
+	'tooltipped-nw': 'tooltipped-sw'
+};
+
+export default function () {
+	for (const el of select.all('.file-header [class*=tooltipped-n]')) {
+		let direction;
```

Comment:
Lo-fi one-liner to replace everything in this loop

```js
el.className = el.className.replace('tooltipped-n', 'tooltipped-s');
```

Reply:
I like how explicit the current implementation is, but I understand that it might be a bit too verbose. 👍 

I'll replace it with your suggestion, and probably add some comments for future maintainers to explain what it's exactly doing, especially that, at a glance, `el.className.replace` looks very close to `el.classList.replace` but it behaves very differently.